exercises/complex-numbers/description.md

```diff
@@ -1,29 +1,95 @@
 # Description
 
-A complex number is a number in the form `a + b * i` where `a` and `b` are real and `i` satisfies `i^2 = -1`.
+A **complex number** is expressed in the form `z = a + b * i`, where:
 
-`a` is called the real part and `b` is called the imaginary part of `z`.
-The conjugate of the number `a + b * i` is the number `a - b * i`.
-The absolute value of a complex number `z = a + b * i` is a real number `|z| = sqrt(a^2 + b^2)`. The square of the absolute value `|z|^2` is the result of multiplication of `z` by its complex conjugate.
+- `a` is the **real part** (a real number),
 
-The sum/difference of two complex numbers involves adding/subtracting their real and imaginary parts separately:
-`(a + i * b) + (c + i * d) = (a + c) + (b + d) * i`,
-`(a + i * b) - (c + i * d) = (a - c) + (b - d) * i`.
+- `b` is the **imaginary part** (also a real number), and
 
-Multiplication result is by definition
-`(a + i * b) * (c + i * d) = (a * c - b * d) + (b * c + a * d) * i`.
+- `i` is the **imaginary unit** satisfying `i^2 = -1`.
 
-The reciprocal of a non-zero complex number is
-`1 / (a + i * b) = a/(a^2 + b^2) - b/(a^2 + b^2) * i`.
+## Key Properties
 
-Dividing a complex number `a + i * b` by another `c + i * d` gives:
-`(a + i * b) / (c + i * d) = (a * c + b * d)/(c^2 + d^2) + (b * c - a * d)/(c^2 + d^2) * i`.
+### Conjugate
 
-Raising e to a complex exponent can be expressed as `e^(a + i * b) = e^a * e^(i * b)`, the last term of which is given by Euler's formula `e^(i * b) = cos(b) + i * sin(b)`.
+The conjugate of the complex number `z = a + b * i` is given by:
 
-Implement the following operations:
+```math
```

Comment:
Does Exercism's website render `math` blocks?

Reply:
I don't think we do

Reply:
@IsaacG, yes, @ErikSchierboom is right! Exercism doesn't render math blocks (as far as I know, since I haven't seen them anywhere), and they don't work in the forum. However, the tests and checks on GitHub require me to specify the block type. I initially thought it would make sense to specify it as a math block, but my original intention was to keep it as plaintext. I didn't specify plaintext initially, though.

Now, I'm wondering whether I should change the block type to plaintext or leave it as is (since Exercism doesn't support math blocks, so I assume it will render it as plain text instead of throwing an error). The reasoning is that if Exercism ever starts supporting math blocks in the future, it might be useful to leave it as is so it renders appropriately.

What do you think?

Reply:
It might be better to change it to 'plaintext' or 'text'. When I preview the file in vscode, I'm getting rendering errors due to the `math` block.

Reply:
Thanks, @tasxatzial for the clarification! I agree with your reasoning that switching to plain text seems like the better option, especially given Exercism’s current limitations with not rendering math blocks. Aesthetically, it makes sense to avoid rendering errors and ensure everything displays cleanly. You’re right that even if math blocks are supported in the future, they may not necessarily be labelled as such. Changing it to plaintext would likely be the safest choice for now. I’ll push the changes ASAP.